engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -678,4 +625,202 @@ public String toString() {
 					+ ", methodValidationConfiguration=" + methodValidationConfiguration + "]";
 		}
 	}
+
+	static class ValidatorFactoryContext {
```

Comment:
Let's call it `ValidatorFactoryScopedContext` for symmetry.

Same for the builder.

Reply:
do you mean:
`ValidatorFactoryContext` -> `ValidatorFactoryScopedContext`
`Builder` -> `ValidatorFactoryScopedContextBuilder` ? 
I followed the same naming for a builder as for example in `MethodValidationConfiguration.Builder`, or am I missing something :smiley:

Reply:
Ah yes, sorry, keep the builder as it is :).